src/panels/lovelace/cards/hui-humidifier-card.ts

```diff
@@ -26,6 +27,18 @@ import { HumidifierCardConfig } from "./types";
 
 @customElement("hui-humidifier-card")
 export class HuiHumidifierCard extends LitElement implements LovelaceCard {
+  // @ts-ignore
+  private _resizeController = new ResizeController(this, {
+    callback: async (entries) => {
+      const container = entries[0]?.target.shadowRoot?.querySelector(
+        ".container"
+      ) as HTMLElement | undefined;
+      if (!container) return;
+      const height = container.clientHeight;
+      container.style.setProperty("--height", `${height}px`);
```

Comment:
Why don't we do this inside render instead? We should avoid having side effects here, we should just store `clientHeight` as the state of the `_resizeController`

Reply:
I did that do avoid re-render but yeah I will update it to avoid side effect 🙂

Reply:
@bramkragten Done!